### PR TITLE
Adding python app to track google analytics

### PR DESCRIPTION
**Pull request by dinagraves**
Creating a python app to log events to Google Analytics.

[Design Doc](https://docs.google.com/document/d/1SkvQbgH4PSnubDj0e7ClkTHZoy7fAIDgd2RHHsBT5JM/edit?ts=5dfbbb4e#)

Was unsure whether or not we should remove the Go folder, or if it should stay there for historical reference?

### CODE REVIEW

**Comment by markmirch** [comment]
1) Let's get rid of the go folder since we'll always have it in version control if we need it
2) Can we update the folder name to something more descriptive, like "ga_handler"
3) The readme should be updated to match the new folder

**Comment by dinagraves** [comment]
Removed the Go folder, renamed the python folder, and updated the README to reference the python app.  Also added a noxfile to run Lint and tests.

File: ga_handler/main.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# limitations under the License.
+
+from flask import Flask, redirect, request
+import sys
+from urllib.parse import urlencode
+from urllib.request import Request, urlopen
+
+
+# GCS Storage Bucket
+STORAGE_BUCKET = "https://storage.googleapis.com/eps/"
```
[the remark this answers is not in the document]
Comment (dinagraves) [reply ga_handler/main.py]: Thanks!  If the environment variable isn't set, should it create a 500 error?  Or should it just log a warning.  The redirect wouldn't work in any case if it's missing the storage bucket.

File: ga_handler/main.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+@app.route("/dl/<episode_key>")
+def redirect_episode_download(episode_key):
+    # Logs event to Google analytics and redirects to episode in GCS
+    range_req = request.headers.get("Range", None)
+    user_agent_str = request.headers.get("User-Agent")
+
+    # Log request in Google Analytics if it's not a range request
+    # or the range starts at the beginning of the file
```
[the remark this answers is not in the document]
Comment (dinagraves) [reply ga_handler/main.py]: Is this too much? 

```
    # Range requests are likely to come when streaming data 
    # and only fetching small pieces at a time. 
    # We only want to log each unique download, 
    # so we log when it's either not a range request 
    # or if the range request is the beginning of the file
```

**Pull request merged**